DropboxSync.UIL/Program.cs

```diff
@@ -11,7 +12,24 @@
     })
     .ConfigureServices(services =>
     {
+        // Settings ApplicationData App's path
+
+        string appData = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
+        string appPath = Path.Join(appData, "OAC-DropboxSync");
```

Comment:
yeah so I guess you can use Path.join instead of "\\"

Reply:
In our context `Path.Combine()` and `Path.Join()` would do the same

Reply:
UPDATE : Path.Join() is used